lib/driver.js

```diff
@@ -943,6 +943,8 @@ class XCUITestDriver extends BaseDriver {
         alwaysMatch: {},
       }
     });
+
+    this.opts.autoLaunch = true;
```

Comment:
why not to change it for launchApp call directly? 

*This launchApp stuff makes so much pain*

Reply:
I didn't want to break the previous behavior for `launchApp` calls without this capability set to false - currently the behavior of `launchApp()` calls `createSession` which always terminates the tested application before relaunching it (and creates a new session). From the WDA side, calling launchApp will not terminate the application if it is running in the background.

Reply:
to me this all looks like a workaround for a workaround. The `launchApp`/`closeApp`/`resetApp` stuff  itself is an ugly thing - we must try to isolate these calls as much as possible, so they could be easily deprecated and removed from future Appium versions